packages/utilities/psammead-styles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-styles",
-  "version": "7.3.1",
+  "version": "8.0.0",
```

Comment:
I don't think I'd go back and change this now but for future reference, I would say this should have been a minor bump rather than major (so 7.4.0 rather than 8.0.0); major bumps should be reserved for breaking changes. When we remove the fonts we are currently using in a subsequent PR that would be a good candidate for a major bump as any users of the package would need to change there code to use the new noto font.

See for more info on semantic versioning: https://semver.org/

Reply:
> Did this change need any font size changes? I remember us discussing that we did not know where that was configured but I don't know where the investigation went?

This font didn't need any font size updates, however the others do. I've also changed the bump from a major to minor as it was a fairly quick thing to do.